Asterisk.2013/Asterisk.NET.Test/Asterisk.NET.Test/Program.cs

```diff
@@ -138,13 +138,13 @@ private static void checkManagerAPI()
 			manager.RegisterUserEventClass(typeof(UserAgentLoginEvent));
 
 			// Add or Remove events
-			manager.UserEvents += new UserEventHandler(dam_UserEvents);
+			manager.UserEvents += dam_UserEvents;

```

Comment:
Not it's not breaking. But now we don't have UserEventHandler deledate. We have 
**public event EventHandler\<UserEvent\> UserEvents**

For better code readability can change this code to 
**manager.UserEvents += new EventHandler\<UserEvent\>(dam_UserEvents);**